modules/identity/pages/rest-api-authorization.adoc

```diff
@@ -395,30 +395,15 @@ edit `web.xml` and comment out the following definitions:
 [discrete]
 ==== Running in debug mode
 
-If debug mode is activated, whenever you update a configuration file or a dynamic check script, the changes take effect immediately.
+To optimize performance in production, Bonita caches the dynamic check scripts for faster subsequent executions.
 
-To activate debug mode, edit `security-config.properties` and set the value of the `security.rest.api.authorizations.check.debug` property to `true`, then restart the application server.
+If debug mode is activated, Bonita will reload the dynamic check scripts each time they are executed. It allows to rapidly validate your script at *development* time.
 
-To deactivate authorization, set the property to `false`, then restart the application server. Debug mode should be deactivated in production, so as not to impact performance.
+To activate debug mode, edit `bonita-platform-sp.properties` and set the value of the `bonita.runtime.authorization.dynamic-check.debug` property to `true`, then restart the application server.
 
-[#migrate]
+Then, each time you change a dynamic check script, simply update it in database using xref:runtime:bonita-platform-setup.adoc#update_platform_conf[Setup tool] (`./setup.sh push`). The script will be reloaded at next execution (next time you call a URL that matches this dynamic rule).
 
-[discrete]
-==== Migration
```

Comment:
@abirembaut I chose to remove this § as it concerns very old versions.
Is it ok for you?